xcode/Subconscious/Shared/Components/Common/Byline/SlashlinkBylineView.swift

```diff
@@ -16,10 +16,12 @@ struct SlashlinkBylineView: View {
     
     var body: some View {
         HStack(spacing: 0) {
+            // Oh my...
             if let petname = petname {
-                Text(verbatim: "@\(petname)")
-                    .fontWeight(.medium)
-                    .foregroundColor(petnameColor)
+                if let petname = Petname(petname) {
```

Comment:
I went with `flatMap` (otherwise you get an option of an option and only unpack one layer). 

```swift
if let petname = petname.flatMap({ string in Petname(string) }) {
}
```


Sidenote: I've always found myself preferring `bind` as a name vs `flatMap` for Monads, except when it comes to sequences that is.